var/spack/repos/builtin/packages/sollve/package.py

```diff
@@ -0,0 +1,242 @@
+# Copyright 2013-2019 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack import *
+
+
+class Sollve(CMakePackage):
+    """The SOLLVE Project aims at scaling OpenMP by leveraging LLVM for exascale
+       performance and portability of applications.  This package provides a
+       collection of Clang/LLVM compilers and an OpenMP runtime library.
+    """
+
+    homepage = 'https://www.bnl.gov/compsci/projects/SOLLVE/'
+    url      = "https://github.com/SOLLVE"
+
+    family = 'compiler'  # Used by lmod
+
+    # NOTE: The debug version of LLVM is an order of magnitude larger than
+    # the release version, and may take up 20-30 GB of space. If you want
+    # to save space, build with `build_type=Release`.
+
+    variant('clang', default=True,
+            description="Build the LLVM C/C++/Objective-C compiler frontend")
+    variant('lldb', default=True, description="Build the LLVM debugger")
+    variant('lld', default=True, description="Build the LLVM linker")
+    variant('internal_unwind', default=True,
+            description="Build the libcxxabi libunwind")
+    variant('polly', default=True,
+            description="Build the LLVM polyhedral optimization plugin, "
+            "only builds for 3.7.0+")
```

Comment:
Does this mean `sollve@3.7.0:`? If so, you can add a conflict like:
```python
conflicts('+polly', when='@:3.6', msg='polly is only available for SOLLVE 3.7.0+')
```

Reply:
Thanks. Actually this sollve package does not support a version < 3.7.0, so we removed this explanation.